core/unit_test/CMakeLists.txt

```diff
@@ -385,7 +386,8 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;OpenACC;HIP;SYCL)
     set(${Tag}_VIEWSUPPORT)
     IF (Kokkos_ENABLE_IMPL_MDSPAN)
       foreach(Name
-        ReferenceCountedAccessor
+        BasicView
+	ReferenceCountedAccessor
         ReferenceCountedDataHandle
```

Comment:
```suggestion
        BasicView
        ReferenceCountedAccessor
        ReferenceCountedDataHandle
```

Reply:
gonna fix this later. We anyway will need to rebase.

Reply:
fixed